modules/performance/extension/siddhi-execution-performance/component/src/main/java/org/wso2/extension/siddhi/execution/throughput/CalculatePerformanceStreamProcessorExtension.java

```diff
@@ -249,18 +255,25 @@ private static int getLogFileSequenceNumber() {
                         + "found " + attributeExpressionExecutors[1].getReturnType());
             }
 
-            if (attributeExpressionLength == 3) {
-                if (attributeExpressionExecutors[2].getReturnType() == Attribute.Type.INT) {
+            if (attributeExpressionExecutors[2].getReturnType() == Attribute.Type.INT) {
+
+            } else {
+                throw new SiddhiAppValidationException("Third parameter expected to be int but "
+                        + "found " + attributeExpressionExecutors[2].getReturnType());
+            }
+
+            if (attributeExpressionLength == 4) {
+                if (attributeExpressionExecutors[3].getReturnType() == Attribute.Type.INT) {
                     recordWindow = (int) ((ConstantExpressionExecutor) attributeExpressionExecutors[2]).getValue();
                 } else {
-                    throw new SiddhiAppValidationException("Third parameter expected to be int but "
-                            + "found " + attributeExpressionExecutors[1].getReturnType());
+                    throw new SiddhiAppValidationException("Fourth parameter expected to be int but "
+                            + "found " + attributeExpressionExecutors[3].getReturnType());
                 }
             }
 
         } else {
             throw new SiddhiAppValidationException("Input parameters for Log can be iijTimeStamp (Long), " +
-                    "type (String), recordwindow (int) but there are " +
+                    "type (String), throughputCount (int),  recordwindow (int) but there are " +
```

Comment:
Is is throughput count or the order of the instance called?

Reply:
How do you get the values for executionType variable?

Reply:
Type determines whether it is "throughput" or "Latency" or "Both" .It is the default parameter which originally in  the extension.i didn't change that.